src/steps/mdx-to-gridtables.ts

```diff
@@ -31,12 +31,16 @@ function getAttribute(node: MdxJsxFlowElement, name: string) {
 }
 
 function getAttributeValue(attr: MdxJsxAttribute | MdxJsxExpressionAttribute, fallback?: string) {
-  return attr && attr.value && typeof attr.value === 'string' ? attr.value : fallback;
+  if (attr && attr.value) {
+    return typeof attr.value === 'string' ? attr.value : attr.value.value;
+  }
+  return fallback;
 }
 
 // Type guard to filter only MdxJsxAttribute objects
-function isMdxJsxAttribute(attribute: any): attribute is MdxJsxAttribute {
-  return attribute && typeof attribute.name === 'string';
+function isMdxJsxAttribute(attribute: MdxJsxAttribute| MdxJsxExpressionAttribute)
+: attribute is MdxJsxAttribute {
+  return attribute != null;
```

Comment:
Removed check because typescript was complaining and MdxJsxAttribute is guaranteed to have `typeof attribute.name === 'string'`